lib/assertions.ts

```diff
@@ -774,11 +774,13 @@ export default class DOMAssertions {
     }
 
     let result = expectedProperties.every(
-      property => computedStyle[property] === expected[property]
+      property => computedStyle.getPropertyValue(property.toString()) === expected[property]
```

Comment:
A `property` seems to be a string already, is the `toString` method needed?

Reply:
I agree that it's a little silly, but it's this sort of nonsense that typescript is very serious about.

Reply:
It seems like this bit is faulty in that case
`let expectedProperties = Object.keys(expected) as CSSStyleDeclarationProperty[];`

Since the `getPropertyValue` method expects a string and not a `CSSStyleDeclarationProperty` I assume.
It's possible that we could get rid of that cast.

Feel free to leave it :+1: